modules/features/referrals/src/main/kotlin/au/com/shiftyjelly/pocketcasts/referrals/ReferralGuestPassCardView.kt

```diff
@@ -38,17 +40,22 @@ fun ReferralGuestPassCardView(
     val cardTitle = stringResource(LR.string.referrals_send_guest_pass_card_title)
     Box(
         modifier = modifier
-            .background(
-                color = ReferralGuestPassCardDefaults.cardBackgroundColor,
-                shape = RoundedCornerShape(ReferralGuestPassCardDefaults.cardRadius),
-            )
+            .clip(RoundedCornerShape(ReferralGuestPassCardDefaults.cardRadius))
             .border(
                 width = 1.dp,
                 color = ReferralGuestPassCardDefaults.cardStrokeColor,
                 shape = RoundedCornerShape(ReferralGuestPassCardDefaults.cardRadius),
             )
             .semantics { contentDescription = cardTitle },
     ) {
+        BoxWithConstraints(
+            modifier = Modifier.blur(ReferralGuestPassCardDefaults.cardBlur),
+        ) {
+            ReferralCardAnimatedBackgroundView(
+                modifier = Modifier
+                    .size(maxWidth, maxHeight),
+            )
+        }
```

Comment:
Is the `BoxWithConstraints` used here in anticipation of something? If not it can be skipped in favor of the `.fillMaxSize()` modifier.

```suggestion
        ReferralCardAnimatedBackgroundView(
            modifier = Modifier
                .blur(ReferralGuestPassCardDefaults.cardBlur)
                .fillMaxSize(),
        )
```

I also think it might be better to move `.blur()` modifier inside `ReferralCardAnimatedBackgroundView`. This way the caller doesn't have to know about applying `.blur()` and all this stuff is handled as an implementation detail.